### PR TITLE
add usage_charge resource and update recurring charge

### PR DESCRIPTION
**Pull request by ch33sybr3ad**
## Description
- adds support for `usage_charge` resource
- adds `#customize` and `#usage_charges` to `recurring_application_charge`

@Shopify/finance @dominiquesr @kevinhughes27 @dylanahsmith

### CODE REVIEW

File: shopify/resources/recurring_application_charge.py [hunk 1]
```diff
@@ -9,6 +10,12 @@ def _get_first_by_status(resources, status):
 
 class RecurringApplicationCharge(ShopifyResource):
 
+    def usage_charges(self):
+        return shopify.UsageCharge.find(recurring_application_charge_id=self.id)
```
Comment (sluggoman) [inline shopify/resources/recurring_application_charge.py]: __init__ is importing usage_charge as UsageCharge why "import shopify" and use the namespaced version?

**Comment by ch33sybr3ad** [comment]
:eyes:? 

@kevinhughes27 @dylanahsmith

**Comment by kevinhughes27** [comment]
I never got the original ping for this.. looking now!

**Comment by kevinhughes27** [comment]
lgtm

**Comment by dominiquesr** [comment]
Looks good to me but I haven't had to modify this before.

**Pull request merged**